### PR TITLE
[navigation]feat: add home icon in left bottom

### PR DESCRIPTION
**Pull request by SuZhou-Joe**
### Description

- [x] Add home icon in left bottom and remove home in left nav as we already have the icon in left bottom.
- [x] Make bottom container scrollable
- [x] Move workspace settings to `manage workspace` category

### Home icon in left bottom
image

### Make left bottom container scrollable
image

### Add workspace settings under manage workspace category
image

## Changelog

- feat: add home icon in left bottom

### Check List

- [x] All tests pass
  - [x] `yarn test:jest`
  - [x] `yarn test:jest_integration`
- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
- [ ] Update [CHANGELOG.md](./../CHANGELOG.md)
- [ ] Commits are signed per the DCO using --signoff

### CODE REVIEW

File: src/core/public/chrome/ui/header/collapsible_nav_group_enabled.tsx [hunk 1]
```diff
@@ -435,6 +435,7 @@ export function CollapsibleNavGroupEnabled({
         <div
           className={classNames({
             'bottom-container': true,
+            'eui-xScroll': true,
```
Comment (AMoo-Miki) [inline src/core/public/chrome/ui/header/collapsible_nav_group_enabled.tsx:438]: How will this look on browser-OS combos that show a scrollbar?

Comment (SuZhou-Joe) [reply src/core/public/chrome/ui/header/collapsible_nav_group_enabled.tsx:438]: I get your point, I thought the class will help to hide the scrollbar, but if not, let me find if there is any pattern code I can follow in OSD.

File: src/plugins/home/public/application/components/home_icon.tsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+ */
+
+import React from 'react';
+import { EuiButtonIcon } from '@elastic/eui';
+import { CoreStart } from 'opensearch-dashboards/public';
+
+export function HomeIcon({ core, appId }: { core: CoreStart; appId: string }) {
+  return (
+    <EuiButtonIcon
+      aria-label="go-to-home"
```
Comment (ruanyl) [inline src/plugins/home/public/application/components/home_icon.tsx:13]: What about saying:
```suggestion
      aria-label="go to application homepage"
```

File: src/core/public/chrome/ui/header/collapsible_nav_group_enabled.tsx (hunk 1 above)
Comment (ruanyl) [reply src/core/public/chrome/ui/header/collapsible_nav_group_enabled.tsx:438]: Seem behavior of adding the utility css class follows the OS settings. Was the intention to hide the scrollbar but allow user to scroll?

Comment (SuZhou-Joe) [reply src/core/public/chrome/ui/header/collapsible_nav_group_enabled.tsx:438]: I added -ms-overflow-style: -ms-autohiding-scrollbar; to the bottom container to make the behavior of scrollbar consistent. Do you think that's good enough?

File: src/plugins/visualize/public/plugin.ts [hunk 3]
```diff
@@ -228,39 +228,48 @@ export class VisualizePlugin
       },
     });
 
+    const titleInLeftNav = i18n.translate('visualize..leftNav.visualizeTitle', {
```
Comment (Hailong-am) [inline src/plugins/visualize/public/plugin.ts:231]: two dots here, typo?

Comment (SuZhou-Joe) [reply src/plugins/visualize/public/plugin.ts]: Thanks for catching this.

File: src/core/public/chrome/ui/header/collapsible_nav_group_enabled.tsx (hunk 1 above)
Comment (ruanyl) [reply src/core/public/chrome/ui/header/collapsible_nav_group_enabled.tsx:438]: I'm fine with it, UX need to figure out how to fit those overflow icons, I think we can optimize this afterwards.

**Review by ruanyl — DISMISSED** [review]

**Review by ruanyl — APPROVED** [review]

**Review by ananzh — APPROVED** [review]

**Comment by AMoo-Miki** [comment]
Manually backported with #7802

**Pull request merged**